cetmix_tower_aws/readme/USAGE.md

```
@@ -0,0 +1,112 @@
+   - Go to `Cetmix Tower > Servers > Servers`
+   - Open the server where you want to run the command
+
+* **Execute Command from Server**
+   - Click the `Run Command` button at the top of the server form
+   - In the popup dialog:
+     - Select your AWS EC2 command from the dropdown
+     - Verify the configuration values (if any need adjustment)
+     - Click `Run` to execute
+
+* **View Command Results**
+   - After execution, the command log will display showing:
```

markdownlint-cli2 (0.17.2)

Unordered list style
Expected: asterisk; Actual: dash

(MD004, ul-style)